firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -241,21 +267,23 @@ void state3_stillness() {
         saveStateChangeOrAlert(3, 2);
         stateHandler = state0_idle;
     }
+    else if (millis() - state3_stillness_timer >= *max_stillness_time) {
```

Comment:
Moved this to before the Duration Alert test because if we have reached the criteria for both a Stillness and a Duration Alert,  I want the Stillness one to be sent first because it's more important.